build-system/tasks/bundle-size.js

```diff
@@ -22,7 +22,7 @@ const log = require('fancy-log');
 const {getStdout} = require('../exec');
 
 const runtimeFile = './dist/v0.js';
-const maxSize = '79.28KB';
+const maxSize = '80.60KB';
```

Comment:
Did the maxSize increase because we are shipping both polyfills in this PR?

Reply:
Yup.